db/migrations/20230831162419_add_repo_fields.up.sql

```diff
@@ -0,0 +1,6 @@
+BEGIN;
+
+alter table repositories add column origin varchar default 'external' not null;
+alter table repositories add column content_type varchar default 'rpm' not null;
```

Comment:
do you want to add "if not exists" to these"?

Reply:
yes!